### DISCUSSION TITLE
[Hackathon Use Case] - <play with financial data>

### DISCUSSION THREAD

**Discussion by maoo** [finos/finos-pmcs#254] [Ideas]
# Play with financial data
_From @chuck-adaptive (Adaptive)_

This sample use case does not define a business problem nor a list of expected features; it does however provide a list of free data sources that you can play with, which may trigger interesting ideas around challenges that could be tackled.

More info on https://github.com/chuck-adaptive/data-source-review

**Comment by TheJuanAndOnly99** [comment]
Closing discussion in preparation [for this years hackathon](https://www.finos.org/hosted-events/2024-10-25-citi-india-hackathon-supported-by-finos). Feel free to reopen and update if its still relevant.
